src/interface/meta.thrift

```diff
@@ -1201,6 +1218,7 @@ service MetaService {
     ExecResp grantRole(1: GrantRoleReq req);
     ExecResp revokeRole(1: RevokeRoleReq req);
     ListUsersResp listUsers(1: ListUsersReq req);
+    DescribeUserResp describeUser(1: DescribeUserReq req);
```

Comment:
This is duplicate with RPC getUserRoles

Reply:
@heroicNeZha should make sure whether these two interfaces do the same thing. If so, plz don't define duplicate interfaces.

Reply:
sure, I will remove it